### PR TITLE
OpenSUSE Leap 42.2 ppc64le Openstack template

### PR DESCRIPTION
**Pull request by ramereth**
This is what I used to create the latest OpenSUSE builds on ppc64le.

### CODE REVIEW

File: packer/opensuse-leap-42.2-ppc64le-openstack.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      "boot_wait": "6s",
+      "disk_size": "3072",
+      "headless": true,
+      "http_directory": "http",
+      "iso_checksum": "9e08ecfcbe2f9e3875eeea3f8f11d92813c51281ae0f23549c68f87fe90c189e",
+      "iso_checksum_type": "sha256",
+      "iso_url": "http://download.opensuse.org/ports/ppc/distribution/leap/42.2/iso/openSUSE-Leap-42.2-DVD-ppc64le-Build0156-Media.iso",
+      "output_directory": "packer-opensuse-leap-42.2-ppc64le-openstack",
+      "shutdown_command": "echo 'opensuse'|sudo -S /sbin/halt -h -p",
+      "ssh_password": "opensuse",
```
Comment (knightsamar) [inline packer/opensuse-leap-42.2-ppc64le-openstack.json:21]: Where can I read more about what this parameter means? I tried looking for this in packer docs but could not find.

File: packer/http/opensuse-leap-42.2/autoinst.xml [hunk 2]
```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+
+<!-- http://doc.opensuse.org/projects/autoyast/configuration.html -->
+
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
```
Comment (knightsamar) [inline packer/http/opensuse-leap-42.2/autoinst.xml:6]: It's interesting that the second URL leads to a 404. I hope the generator is up-to-date

File: packer/http/opensuse-leap-42.2/autoinst.xml [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        <expire/>
+        <flag/>
+        <inact>-1</inact>
+        <max>99999</max>
+        <min>0</min>
+        <warn>7</warn>
+      </password_settings>
+      <shell>/bin/bash</shell>
+      <uid>1000</uid>
+      <user_password>opensuse</user_password>
```
Comment (knightsamar) [inline packer/http/opensuse-leap-42.2/autoinst.xml:201]: Does this mean default  password for opensuse user is opensuse?

File: packer/opensuse-leap-42.2-ppc64le-openstack.json (hunk 1 above)
Comment (ramereth) [reply packer/opensuse-leap-42.2-ppc64le-openstack.json:21]: This is actual password packer uses to ssh into the host during the script phase.

File: packer/http/opensuse-leap-42.2/autoinst.xml (hunk 2 above)
Comment (ramereth) [reply packer/http/opensuse-leap-42.2/autoinst.xml:6]: I used the generate feature from opensuse on the host.

File: packer/http/opensuse-leap-42.2/autoinst.xml (hunk 3 above)
Comment (ramereth) [reply packer/http/opensuse-leap-42.2/autoinst.xml:201]: Yes, but cloud-init will lock the user on boot. This follows the same standard we do on all of the other vms.

**Review by knightsamar — APPROVED** [review]

**Pull request merged**